<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AT106HSettings</class>
 <widget class="QDialog" name="AT106HSettings">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>267</width>
    <height>287</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AT106H Settings</string>
  </property>
  <layout class="QFormLayout" name="formLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Resolution</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QComboBox" name="ResComboBox">
     <item>
      <property name="text">
       <string>0.001 mg</string>
      </property>
     </item>
     <item>
      <property name="text">
       <string>0.01 mg</string>
      </property>
     </item>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Centerings</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLineEdit" name="CenteringsEdit"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Wait time (hrs)</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="WaitTimeEdit"/>
   </item>
   <item row="3" column="0">
    <widget class="QLabel" name="label_6">
     <property name="text">
      <string>Stabilization time (s)</string>
     </property>
    </widget>
   </item>
   <item row="3" column="1">
    <widget class="QLineEdit" name="StabTimeEdit"/>
   </item>
   <item row="5" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Pre-Runs</string>
     </property>
    </widget>
   </item>
   <item row="5" column="1">
    <widget class="QLineEdit" name="PreRunsEdit"/>
   </item>
   <item row="6" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Runs</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QLineEdit" name="RunsEdit"/>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="TareCheckBox">
     <property name="text">
      <string>Tare at start</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="AdjustCheckBox">
     <property name="text">
      <string>Adjustment between runs</string>
     </property>
    </widget>
   </item>
   <item row="9" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>144</width>
       <height>12</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="10" column="0" colspan="2">
    <widget class="QPushButton" name="SubmitButton">
     <property name="text">
      <string>Submit</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_7">
     <property name="text">
      <string>Integration time (s)</string>
     </property>
    </widget>
   </item>
   <item row="4" column="1">
    <widget class="QLineEdit" name="IntegrationEdit"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
